### PR TITLE
YPRO-78 Banner support

### PR DESCRIPTION
**Pull request by pabloserranof**
Cleanup and minor refactoring to make the banner works

### CODE REVIEW

File: app/src/main/res/layout/activity_banner.xml [hunk 1]
```diff
@@ -2,14 +2,22 @@
...
         android:gravity="center"
+        android:text="Hola"
         android:textAppearance="?android:attr/textAppearanceLarge" />
 
+    <RelativeLayout
+        android:layout_below="@+id/name"
+        android:background="@color/colorPrimary"
+        android:id="@+id/internal_view"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"></RelativeLayout>
```
Comment (dzolnai) [inline app/src/main/res/layout/activity_banner.xml]: You can use self closing tag

File: mraid_library/src/main/java/org/nexage/sourcekit/mraid/MRAIDView.java [hunk 2]
```diff
@@ -385,51 +297,15 @@ public boolean onTouch(View v, MotionEvent event) {
...
-        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN_MR1) {
-            wv.getSettings().setMediaPlaybackRequiresUserGesture(false);
-        }
-
-        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.KITKAT) {
-            if ((context.getApplicationInfo().flags & ApplicationInfo.FLAG_DEBUGGABLE) != 0) {
-                WebView.setWebContentsDebuggingEnabled(true);
-            }
-        }
+//		if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.KITKAT) {
```
Comment (dzolnai) [inline mraid_library/src/main/java/org/nexage/sourcekit/mraid/MRAIDView.java]: Add comment why it is commented out. If not needed, remove

File: mraid_library/src/main/java/org/nexage/sourcekit/mraid/MRAIDView.java [hunk 3]
```diff
@@ -451,119 +327,75 @@ public void clearView() {
...
-                    getClass().getDeclaredMethod(command).invoke(this);
-                } catch (NoSuchMethodException e) {
-                    getClass().getSuperclass().getDeclaredMethod(command).invoke(this);
-                }
-            } else if (Arrays.asList(COMMANDS_WITH_STRING).contains(command)) {
+            if (Arrays.asList(commandsWithNoParam).contains(command)) {
+                Method method = getClass().getDeclaredMethod(command);
+                method.invoke(this);
+            } else if (Arrays.asList(commandsWithString).contains(command)) {
+                Method method = getClass().getDeclaredMethod(command, String.class);
```
Comment (dzolnai) [inline mraid_library/src/main/java/org/nexage/sourcekit/mraid/MRAIDView.java]: Why do we use reflection here? This is unsafe and slow

File: mraid_library/src/main/java/org/nexage/sourcekit/mraid/MRAIDView.java [hunk 4]
```diff
@@ -1313,25 +1151,17 @@ public boolean onConsoleMessage(ConsoleMessage cm) {
...
-            MRAIDLog.d("hz-m MRAIDView ChromeClient - onJsBeforeUnload");
-            return true;
-        }
-
-        @Override
-        public boolean onJsAlert(WebView view, String url, String message, JsResult result) {
-            MRAIDLog.d("JS alert", message);
-            return handlePopups(result);
-        }
+//		@Override
```
Comment (dzolnai) [inline mraid_library/src/main/java/org/nexage/sourcekit/mraid/MRAIDView.java]: Add reason why it is commented

File: mraid_library/src/main/java/org/nexage/sourcekit/mraid/MRAIDView.java (hunk 3 above)
Comment (pabloserranof) [reply mraid_library/src/main/java/org/nexage/sourcekit/mraid/MRAIDView.java]: I agree with that. However, this is the way the library is implemented. Not in the scope at the moment to refactor this or in this PR

File: app/src/main/res/layout/activity_banner.xml (hunk 1 above)
Comment (pabloserranof) [reply app/src/main/res/layout/activity_banner.xml]: ok

File: mraid_library/src/main/java/org/nexage/sourcekit/mraid/MRAIDView.java (hunk 4 above)
Comment (pabloserranof) [reply mraid_library/src/main/java/org/nexage/sourcekit/mraid/MRAIDView.java]: ok

**Pull request merged**